spark/src/main/scala/io/ddf/spark/etl/DateUDFs.scala

```diff
@@ -112,6 +112,17 @@ object DateUDF {
     }
   }
 
+  val parseWeekOfMonth: Object => Integer = {
+    (obj: Object) => {
+      val dateTime = Utils.toDateTimeObject(obj)
+      if(dateTime != null) {
+        (dateTime.getDayOfMonth() / 7) + 1
```

Comment:
How useful is this compared to the `weekofyear` approach (week belongs to a month if more of it falls in that month)?